### PR TITLE
Issue: #88 Fix compliance on unassigned profile fails

### PR DESCRIPTION
**Pull request by jonmccormick**

### CODE REVIEW

File: src/listener/server-profile-templates.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ copies of the Software, and to permit persons to whom the Software is
 furnished to do so, subject to the following conditions:
 
 The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
+all copies or substMakeProfileCompliantantial portions of the Software.
```
Comment (danielreed) [inline src/listener/server-profile-templates.js]: You'll have to fix this.  We can't modify the license text in any of the files.

File: test/listener/server-profile-templates.js [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+      serverProfileTemplateUri: "/rest/server-profile-templates/650449e6",
+      templateCompliance: "Compliant",
+      serverHardwareUri: null,
+  };
+
+  const profilesResponse = {
+      members: [profileResponse]
+   };
+
+let listeners = [];
```
Comment (danielreed) [inline test/listener/server-profile-templates.js]: does this need indent?

File: test/listener/server-profile-templates.js [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+        new PromiseFeedback((feedback) => {
+            return Bluebird.resolve(profileResponse);
+        })
+    );
+
+    serverProfileTemplateListener.MakeProfileCompliant(profile, msg);
+
+    assert(serverProfileTemplateListener.PowerOffHardware.callCount === 0);
+    assert(serverProfilesListener.MakeServerProfileCompliant.callCount === 1);
+    //console.log(serverProfilesListener.MakeServerProfileCompliant);
```
Comment (danielreed) [inline test/listener/server-profile-templates.js]: remove

File: test/listener/server-profile-templates.js [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+            return Bluebird.resolve(profileResponse);
+        })
+    );
+
+    serverProfileTemplateListener.MakeProfileCompliant(profile, msg);
+
+    assert(sp1.calledOnce);
+
+    sp1.restore();
+
```
Comment (danielreed) [inline test/listener/server-profile-templates.js]: probably want to call restore on st1 and st2

Comment (jonmccormick) [reply test/listener/server-profile-templates.js]: I tried to.  It threw an exception

**Pull request merged**